distros/iota/gradle.properties

```diff
@@ -1,4 +1,4 @@
 # The Iota distro is for only fundamental modules, what you might expect in a "basic" game (a "thin" distribution")
 # Previously a "Mode Zero" concept has been discussed on a related note but this may be less name confusing?
-extraModules=Breathing,Hunger,Malicious,Minimap,NameGenerator,Pathfinding,Sample,Spawning,StructuralResources
-zipFileName=TerasologyIota.zip
\ No newline at end of file
+extraModules=BiomesAPI,CoreAdvancedAssets,CoreAssets,CoreSampleGameplay,CoreWorlds,Drops,Explosives,Furnishings,Health,Inventory,ModuleTestingEnvironment
```

Comment:
Should the MTE be in here? Is _Iota_ a user-facing concept or a dev/devOps concept?

Reply:
Until we come up with test-only dependencies in modules the MTE is a hard dependency for Health.

Iota was thought mostly as a modder-centric concept: Start with this stuff for the basics. In which case in theory CoreSampleGameplay wouldn't fit in here - Iota isn't really playable, if that makes sense. Although it could be a worthwhile convenience to pair it that way anyway (include a gameplay module that makes up the _vanilla_ way to play the game)

I'm not sure there's much of a point to shipping an Iota zip - it would be smaller than Omega, but both are relatively small in the grand scheme of things. Maybe something minimal for modders to rely on in a future where you can mod right out of a game zip somehow?

Reply:
Isn't MTE an _optional_ dependency? See https://github.com/Terasology/Health/blob/develop/module.txt#L9

Reply:
Oh neat! Didn't realize it had been switched over :-)

In that case yeah, we could probably do without 👍 

Bumped into https://github.com/MovingBlocks/Terasology/issues/3981 while poking around that topic